settings.yaml

```diff
@@ -25,6 +25,9 @@ llm:
 embedding:
   # Should be matching the value above in most cases
   mode: local
+  ingest_mode: simple
+#  ingest_mode: parallel
```

Comment:
I'd try not to have commented out entries in settings.yaml. Maybe a comment pointing to the documentation section would be useful though.

Reply:
Done